ccdproc/image_collection.py

```diff
@@ -689,16 +689,19 @@ def _generator(self, return_type,
 
         for full_path in self._paths():
             no_scale = do_not_scale_image_data
-            hdulist = fits.open(full_path,
-                                do_not_scale_image_data=no_scale)
+            if return_type == 'ccd':
+                ccddata = fits_ccddata_reader(full_path, **ccd_kwargs)
```

Comment:
Should `do_not_scale_image_data` be added to `ccd_kwargs`? It definitely wasn't before, so maybe not...but right now if it gets passed in to the generator it will be ignored for the `ccds` generator, I think.